integration_tests/configs/utils.jsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  validator(mnemonic):: {
+    coins: '1000000000000000000stake,10000000000000000000000basetcro',
+    staked: '1000000000000000000stake',
+    mnemonic: mnemonic,
+  },
+  validator_with_timeout(mnemonic):: self.validator(mnemonic) {
+    config: {
```

Comment:
it's supported to set common configs in upper level now: https://github.com/crypto-com/pystarport/pull/56

Reply:
do you mean merge with validator instead of replace whole validators in jsonnet?